addons/event_crm/tests/test_event_crm_flow.py

```diff
@@ -1,12 +1,14 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
+from datetime import timedelta
 from unittest.mock import patch
 
 from odoo.addons.base.tests.test_ir_cron import CronMixinCase
 from odoo.addons.event_crm.tests.common import TestEventCrmCommon
 from odoo.tests import tagged
 from odoo.tests.common import users
+from odoo import fields
```

Comment:
we could probably use the datetime lib instead (minus one line :slightly_smiling_face: )